### PR TITLE
Typing grid rows stores

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Typing grid rows stores

### CODE REVIEW

File: packages/frontend-core/src/components/grid/stores/datasources/nonPlus.ts [hunk 1]
```diff
@@ -97,34 +90,5 @@ export const initialise = (context: StoreContext) => {
...
-        const $fetch = get(fetch)
-        if (!isSameDatasource($fetch?.options?.datasource, $datasource)) {
-          return
-        }
-        $fetch.update({
-          sortOrder: $sort.order || SortOrder.ASCENDING,
-          sortColumn: $sort.column,
-        })
-      })
-    )
```
Comment (aptkingston) [inline packages/frontend-core/src/components/grid/stores/datasources/nonPlus.ts:128]: These are needed for inline filtering and sorting to work - what was the reason for removing them?

Normal usage of these features:
- Right click column header and change sorting
- Seaching via inline filtering (hovering over the column header and clicking the search icon, then entering a value)

File: packages/frontend-core/src/components/grid/stores/datasources/table.ts [hunk 2]
```diff
@@ -108,34 +112,5 @@ export const initialise = (context: StoreContext) => {
...
-        const $fetch = get(fetch)
-        if ($fetch?.options?.datasource?.tableId !== $datasource.tableId) {
-          return
-        }
-        $fetch.update({
-          sortOrder: $sort.order || SortOrder.ASCENDING,
-          sortColumn: $sort.column,
-        })
-      })
-    )
```
Comment (aptkingston) [inline packages/frontend-core/src/components/grid/stores/datasources/table.ts:139]: Same as above.

File: packages/frontend-core/src/components/grid/stores/datasources/viewV2.ts [hunk 3]
```diff
@@ -207,38 +201,6 @@ export const initialise = (context: StoreContext) => {
...
-        }
-        const $fetch = get(fetch)
-        if ($fetch?.options?.datasource?.id !== $datasource.id) {
-          return
-        }
-        $fetch.update({
-          filter: $allFilters,
-        })
-      })
-    )
```
Comment (aptkingston) [inline packages/frontend-core/src/components/grid/stores/datasources/viewV2.ts:240]: Same as above.

**Review by aptkingston — CHANGES_REQUESTED** [review]
LGTM other than the removal of code required for sorting and inline searching.

File: packages/frontend-core/src/components/grid/stores/datasources/nonPlus.ts (hunk 1 above)
Comment (adrinr) [reply packages/frontend-core/src/components/grid/stores/datasources/nonPlus.ts]: I totally missed that. I actually wanted to ask you, as I did not see the actual usage of that, and I thought it was some legacy code. Fixed!

**Review by aptkingston — APPROVED** [review]

**Pull request merged**